index.bs

```diff
@@ -203,8 +230,27 @@ spec: mediaqueries-4; urlPrefix: https://drafts.csswg.org/mediaqueries-4/#
       </p>
 
       <p>
-        A <dfn>valid VideoConfiguration contentType</dfn> MUST include a media
-        type, subtype and full codecs description.
+        To check if a {{VideoConfiguration}} <var>configuration</var> is a
+        <dfn>valid video configuration</dfn>, the following steps MUST be run:
+        <ol>
+          <li>
+            If <var>configuration</var> is <code>null</code>, return
+            <code>true</code> and abort these steps.
+          </li>
+          <li>
+            If <var>configuration</var>'s {{VideoConfiguration/contentType}} is
```

Comment:
That is the meaning I took. As in, does the platform support it, which will naturally be "false" for any invalid mime string